doc/_templates/layout.html

```diff
@@ -0,0 +1,19 @@
+{% extends "!layout.html" %}
+
+{%- block extrahead %}
+
+   {% if theme_googletag  %}
```

Comment:
I talked to Bernie and he pointed out that we want the default to be false. It doesn't make sense for local docs builds to include it.

We'll only enable this for the production builds on http://api.mongodb.com/python.